tests/lib/lte_lc_api/src/functions_to_mock.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
We should avoid creating this new file. Lets try this together when I am back in the office on Friday. :-)